quanti_gin/data_generator.py

```diff
@@ -448,13 +529,21 @@ def main():
 
     job_generator = DataGenerator.generate_jobs
 
+    if number_of_atoms >= 10 and args.fidelity == True:
+        warnings.warn(
+            "The calculations may be very slow due to the high number of atoms and high fidelity setting.",
```

Comment:
```suggestion
            "The calculations may be very slow due to the high number of atoms, and enabled fidelity calculation .",
```

Reply:
Would change the warning as there is no "high" fidelity setting, rather just a toggle switch?